build-conflicts.php

```diff
@@ -83,8 +84,8 @@ static function (int $errorCode, string $message = '', string $file = '', int $l
         E_STRICT | E_NOTICE | E_WARNING
     );
 
-    $token                     = getenv('GITHUB_TOKEN') ?: '';
-    $authentication            = $token === '' ? '' : $token . ':x-oauth-basic@';
+    $token                     = non_empty_string()->coerce(get_var('GITHUB_TOKEN'));
```

Comment:
note! `non_empty_string()->coerce($var)` will throw if $var is null ( which in this case can be since `get_var` returns `null` if the variable is not defined.

Reply:
That's expected/wanted here: before, I had an assertion way later in the code, now moved here.